src/commands/reply.js

```diff
@@ -1,13 +1,37 @@
 const debug = require('debug');
 const log = debug('Reply');
 
+const dickbutt_img = 'https://i.imgur.com/etjgJ2D.jpg';
+
+const greetings = [
+  'hi',
+  'sup',
+  '~ohayo'
+];
+
+function randomGreeting() {
+  greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  log(greeting);
+  return greeting;	
+}
+
 const responses = new Map();
 responses.set('ping', 'pong');
+responses.set('sing', 'song');
+//this creates the response at startup time
+//want to dynamically create response
+responses.set('hi bot', randomGreeting());
```

Comment:
It only sets to one greeting when the bot starts up. Don't merge this part in yet. It'll be obsolete when we reimplement reply.js anyway.